backend/internal/service/handler/media/get_media.go

```diff
@@ -23,6 +27,24 @@ func (h *Handler) GetMediaByName(c *fiber.Ctx) error {
 		mediaType = models.BothMedia
 	}
 	medias, _ := h.mediaRepository.GetMediaByName(c.Context(), name, mediaType)
+	log.Println(medias)
```

Comment:
i think changes from your other branch snuck into here :( can u remove them for the time being while we figure out that other PR?

Reply:
very true, removed!